src/resources/views/admin/purchases/show.blade.php

```diff
@@ -161,6 +161,24 @@
 
 			</div>
 		</div>
+
```

Comment:
> Also, if the purchased ticket count is not equal to the participant count, there should be a message.
In the best case those two cases should somehow also be visible in the purchases index blade, so changed purchases can be seen in the overview.

Saw/fully understood that part only later.  
From what I can tell the number entered during the purchasing process only determines how many `EventParticipants` get created. It is not actually stored along with the `Purchase`. So this would not actually work, without also changing the `Purchase` model. We could do that, but given that the whole shop system is supposed to be reworked, I'd move that part of the issue to then.

Reply:
Thanks for the additions :) Sadly i did not mean any indication in the row, but thats also a good addition. But i'm going to do the addition that i intended :)